client/js/analysis.js

```diff
@@ -64,7 +64,7 @@ function addFeedbackResult(result) {
 function addDonutChart(str, percent) {
   const div = document.createElement('div');
 
-  div.setAttribute('id', 'a-chart');
+  div.classList = 'a-chart';
   div.className = 'item donut';
```

Comment:
Using div.className will clear any classes applied to the div, so a-chart will no longer exist in its classList. You should add a-chart along with item and donut in the className assignment (classList.add would be used if you wanted to add these classes without affecting any other classes that might be applied to the div already).